docs/src/app/components/master.jsx

```diff
@@ -21,26 +21,22 @@ const ThemeManager = Styles.ThemeManager;
 const DefaultRawTheme = Styles.LightRawTheme;
 
 
-const Master = React.createClass({
-  getInitialState () {
-    return {
+class Master extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this._handleTabChange = this._handleTabChange.bind(this);
```

Comment:
I'm wondering. What is the advantage of using the `bind` operator?
This pattern seems to make the code maintainability harder.
Couldn't we use an arrow function or an `@autobind`?